src/main/java/net/minecraftforge/client/model/obj/ObjModel.java

```diff
@@ -280,6 +282,21 @@ static ObjModel parse(ObjTokenizer tokenizer, ModelSettings settings, Map<String
                 case "o":
                 {
                     String name = line[1];
+                    if (model.parts.containsKey(name))
+                    {
+                        hasHomonymObjectNameAppeared = true;
+                        if (allowHomonymObjectName)
+                        {
+                            int suffixNum = 0;
```

Comment:
I wonder if we should begin this at 1 instead, so the names would be: `object`, `object_1`, `object_2`.